2021-06-18 03:59:17,617 INFO    MainThread:4026006 [internal.py:wandb_internal():91] W&B internal server running at pid: 4026006, started at: 2021-06-18 03:59:17.616829
2021-06-18 03:59:17,626 DEBUG   HandlerThread:4026006 [handler.py:handle_request():124] handle_request: run_start
2021-06-18 03:59:17,631 INFO    WriterThread:4026006 [datastore.py:open_for_write():80] open: /home/chengzilong/DDQN/wandb/offline-run-20210618_035915-1lc1k2ql/run-1lc1k2ql.wandb
2021-06-18 03:59:19,966 DEBUG   HandlerThread:4026006 [meta.py:__init__():39] meta init
2021-06-18 03:59:19,967 DEBUG   HandlerThread:4026006 [meta.py:__init__():53] meta init done
2021-06-18 03:59:19,967 DEBUG   HandlerThread:4026006 [meta.py:probe():210] probe
2021-06-18 03:59:19,973 DEBUG   HandlerThread:4026006 [git.py:repo():33] git repository is invalid
2021-06-18 03:59:19,974 DEBUG   HandlerThread:4026006 [meta.py:_save_pip():57] save pip
2021-06-18 03:59:19,976 DEBUG   HandlerThread:4026006 [meta.py:_save_pip():71] save pip done
2021-06-18 03:59:19,976 DEBUG   HandlerThread:4026006 [meta.py:_save_conda():78] save conda
2021-06-18 03:59:24,043 DEBUG   HandlerThread:4026006 [meta.py:_save_conda():86] save conda done
2021-06-18 03:59:24,044 DEBUG   HandlerThread:4026006 [meta.py:probe():252] probe done
